src/contract/mod.rs

```diff
@@ -382,18 +445,19 @@ impl Document {
         key: &str,
         document_type_name: &str,
         contract: &Contract,
+        owner_id: Option<&[u8]>,
     ) -> Result<Option<Vec<u8>>, Error> {
-        match self.properties.get(key) {
-            None => Ok(None),
-            Some(value) => {
-                let document_type =
-                    contract
-                        .document_types
-                        .get(document_type_name)
-                        .ok_or(Error::CorruptedData(String::from(
-                            "document type should exist for name",
-                        )))?;
-                Ok(Some(document_type.serialize_value_for_key(key, value)?))
+        if key == "$ownerId" && owner_id.is_some() {
+            Ok(Some(Vec::from(owner_id.unwrap())))
```

Comment:
this is a redundant unwrap. can you do smth like
```suggestion
        if key == "$ownerId" && let Some(owner_id) = unwrapped_owner_id {
            Ok(Some(Vec::from(unwrapped_owner_id)))
```

Reply:
nevermind, this isn't easy to do here :)